l10n_it_central_journal/readme/USAGE.rst

```diff
@@ -0,0 +1,23 @@
+**Italiano**
+
+Molto spesso non è possibile stampare un anno intero in un unico PDF per i limiti del tool di stampa di Odoo.
+Quindi è necessario stampare il linro giornale suddividendolo in periodi più brevi  ad esempio un mese alla volta.
+Da normativa quando si suddivide in più periodi bisogna indicare i saldi di riporto tra i vari periodi.
+Al momento per poter riportare i saldi tra i vari periodi è necessario fare queste operazioni.
+
+Creare dei periodi appositi per il libro giornale.
+contabilità o fatturazione -> configurazione -> intervalli data .
+ad esempio "libro giornale gennaio 2020", "libro giornale febbraio 2020", ecc
+
+Stampare il libro giornale del mese di gennaio con intervallo data "libro giornale gennaio 2020" con il button la stampa definitiva.
+In questo modo nell'intervallo data "libro giornale gennaio 2020" verranno indicati i progressivi dare avere di gennaio.
+Copiare questi progressivi nell'intervallo data "libro giornale febbraio 2020" e poi stampare il libro giornale in stampa definitiva.
+Il report di Febbraio avrà i saldi iniziali indicati in precedenza che con la stampa definitiva vengono aggiornati con i nuovi progressivi di fine febbraio.
+Ripetere l'operazione per tutti i periodi
+
+Il numero di pagina anch'esso è da riportare a mano in fase di stampa. Questo perchè il report viene generato in html e poi trasformato in PDF e quindi non è possibile stabilire apriori quante pagine verranno generate.
+
+**English**
+
+Same time is not possible to print the general ledger in one PDF file this due to the report tool of odoo.
+To solve print smaller periods eg one month by time.
```

Comment:
Se non hai intenzione di tradurre tutto il testo, forse è meglio lasciare solo la versione italiana.

Reply:
come discusso in call non ha senso perdere tempo in una traduzione che non servirà a nessuno, dubito che uno straniero sia interessato. Ho lasciato giusto un minimo per far capire di cosa si tratta, poi se uno straniero è interessato c'è google traslate.

Reply:
Ok però questa frase non è corretta. Se vuoi dirci la frase in italiano proviamo a tradurla.

Aggiungerei comunque "See the Italian section for more extensive information"

Reply:
Altrimenti la parte inglese potremmo toglierla direttamente